common/exception/ErrorMessage.java

```diff
@@ -28,40 +28,42 @@ private ErrorMessage(String codePrefix, int codeNumber, String messagePrefix, St
     }
 
     public static class Client extends ErrorMessage {
+        public static final Client RPC_METHOD_UNAVAILABLE =
+                new Client(1, "The server does not support this method, please check the client-server compatibility:\n'%s'.");
```

Comment:
This error message could be friendlier, something like
"The server does not support this method. Please ensure that the TypeDB Client and TypeDB Server versions are compatible ..."

Reply:
Will push an update, thank you!